kubejs/server_scripts/enigmatica/kubejs/base/recipes/shaped.js

```diff
@@ -218,44 +218,83 @@ events.listen('recipes', (event) => {
             }
         },
         {// Add Oak Recipes for Forest Combs
-            output: Item.of("minecraft:oak_log", 16),
+            output: Item.of("minecraft:oak_log", 8),
             pattern: ['BCB', 'C C', 'BCB'],
             key: {
-                B: 'resourcefulbees:forest_honeycomb',
-                C: 'resourcefulbees:wax'
+                C: 'resourcefulbees:forest_honeycomb',
+                B: 'resourcefulbees:wax'
             }
         },
         {
-            output: Item.of("minecraft:oak_sapling", 8),
+            output: Item.of("minecraft:oak_sapling", 4),
             pattern: [' C ', 'B B', ' C '],
             key: {
-                B: 'resourcefulbees:forest_honeycomb',
-                C: 'resourcefulbees:wax'
+                C: 'resourcefulbees:forest_honeycomb',
+                B: 'resourcefulbees:wax'
             }
         },
         {
             output: Item.of("minecraft:oak_leaves", 16),
             pattern: ['   ', 'B C', '   '],
             key: {
-                B: 'resourcefulbees:forest_honeycomb',
-                C: 'resourcefulbees:wax'
+                C: 'resourcefulbees:forest_honeycomb',
+                B: 'resourcefulbees:wax'
             }
         },
         {// Add Stone Recipes for Rocky Combs
             output: Item.of("minecraft:stone", 32),
             pattern: ['BCB', 'CAC', 'BCB'],
             key: {
                 A: ['#forge:storage_blocks/coal','#forge:storage_blocks/charcoal','#forge:storage_blocks/coal_coke','minecraft:lava_bucket'],
-                B: 'resourcefulbees:rocky_honeycomb',
-                C: 'resourcefulbees:wax'
+                C: 'resourcefulbees:rocky_honeycomb',
+                B: 'resourcefulbees:wax'
             }
         },
         {
             output: Item.of("minecraft:cobblestone", 32),
             pattern: ['BCB', 'C C', 'BCB'],
             key: {
+                C: 'resourcefulbees:rocky_honeycomb',
+                B: 'resourcefulbees:wax'
+            }
+        },
+        {
+            output: Item.of("minecraft:diorite", 32),
+            pattern: ['CBC', 'DAD', 'CBC'],
+            key: {
+                A: ['#forge:storage_blocks/coal','#forge:storage_blocks/charcoal','#forge:storage_blocks/coal_coke','minecraft:lava_bucket'],
+                B: 'resourcefulbees:rocky_honeycomb',
+                C: 'resourcefulbees:wax',
+                D: 'resourcefulbees:nether_quartz_honeycomb'
+            }
+        },
+        {
+            output: Item.of("minecraft:andesite", 32),
+            pattern: ['CDC', 'BAB', 'CDC'],
+            key: {
+                A: ['#forge:storage_blocks/coal','#forge:storage_blocks/charcoal','#forge:storage_blocks/coal_coke','minecraft:lava_bucket'],
                 B: 'resourcefulbees:rocky_honeycomb',
-                C: 'resourcefulbees:wax'
+                C: 'resourcefulbees:wax',
+                D: 'resourcefulbees:nether_quartz_honeycomb'

```

Comment:
I think this is too expensive, perhaps remove the Coal Block and decrease output to 24?